pkgs/development/libraries/lcms/default.nix

```diff
@@ -1,10 +1,11 @@
 {stdenv, fetchurl}:
 
-stdenv.mkDerivation {
-  name = "lcms-1.19";
+stdenv.mkDerivation rec {
+  pname = "lcms";
+  version = "1.19";
 
   src = fetchurl {
-    url = "http://www.littlecms.com/lcms-1.19.tar.gz";
+    url = "https://downloads.sourceforge.net/project/${pname}/${pname}/${version}/${pname}-${version}.tar.gz";
```

Comment:
If it works the format should be: ``mirror://sourceforge/lcms/${pname}-${version}.tar.gz";``.

Reply:
Updated.  Thanks!